x/gamm/keeper/msg_server_test.go

```diff
@@ -77,7 +77,7 @@ func (s *KeeperTestSuite) TestSwapExactAmountIn_Events() {
 			},
 			tokenIn:               sdk.NewCoin(doesNotExistDenom, sdk.NewInt(tokenIn)),
 			tokenOutMinAmount:     sdk.NewInt(tokenInMinAmount),
-			expectedMessageEvents: 1, // 1 event gets triggered prior to failure.
+			expectedMessageEvents: 2, // 2 event gets triggered prior to failure.
```

Comment:
Adam and I were unable to determine what the additional triggered event was here. The only running theory we have is that previously this swap would get caught on insufficient output, which became changed after this PR set the default taker fee to zero, but this is not particularly convincing. cc @p0mvn